index.html

```diff
@@ -239,29 +227,76 @@ <h3 class="title">Skills</h3>
         </section>
         <section id="contact" class="contact-form">
             <h2 class="primary-text">
-                I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.
+                I'm always <br>
+                 interested in <br>
+                 hearing about <br> 
+                 new projects, so <br>
+                  if you'd like to <br> 
+                 chat please get <br> 
+                 in touch.
             </h2>
-            <form method="post" autocomplete="off"  class="input-form">
-                 <input id="name-input"  class="full-name"  type="text"  placeholder="Full name"  name="full-name" />
-                <input id="email-input" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" type="email" placeholder="Email address" name="email"/>
-                <textarea id="textarea-input" class="details" id="details" placeholder="Hey I’d like to connect to..."  name="message" ></textarea>
-                <button id="submitBtn" name="submit" class="green-btn-submit" type="submit" title="press" /> Get in touch </button>
-                     <p class="contact-email">yesuf023@gmail.com</p>
-            </form>
+            <form
+            autocomplete="off"
+        action="https://formspree.io/f/{form_id}" method="post"
```

Comment:
- [ ] Kindly head to [formspree services](https://formspree.io/html/), create an account to get your `form_id` and use the id in place `{form_id}`. With that, you form form can be submitted with `404` error.

### Your form
![your-form](https://github.com/yesuf023/Portfolio/assets/26597432/aa30f0d6-6cf6-4d87-9909-cb5f699b7c03)